sklearn/metrics/regression.py

```diff
@@ -272,7 +276,15 @@ def median_absolute_error(y_true, y_pred):
                                                    'uniform_average')
     if y_type == 'continuous-multioutput':
         raise ValueError("Multioutput not supported in median_absolute_error")
-    return np.median(np.abs(y_pred - y_true))
+    if sample_weight is None:
+        return np.median(np.abs(y_pred - y_true))
```

Comment:
can we do `sample_weight = np.ones_like(y_true)` here and pass `sample_weight` to weighted median as a sanity check?